config/defaults.mk

```diff
@@ -510,12 +510,12 @@ endif
 ifdef CAMP_DIR
    RAJA_OPT += -I$(CAMP_DIR)/include
 endif
-RAJA_LIB = $(XLINKER)-rpath,$(RAJA_DIR)/lib -L$(RAJA_DIR)/lib -lRAJA
+RAJA_LIB = $(XLINKER)-rpath,$(RAJA_DIR)/lib -L$(RAJA_DIR)/lib -lRAJA -lcamp
 
 # UMPIRE library configuration
 UMPIRE_DIR = @MFEM_DIR@/../umpire
 UMPIRE_OPT = -I$(UMPIRE_DIR)/include $(if $(CAMP_DIR), -I$(CAMP_DIR)/include)
-UMPIRE_LIB = -L$(UMPIRE_DIR)/lib -lumpire
+UMPIRE_LIB = -L$(UMPIRE_DIR)/lib -lumpire -lcamp
```

Comment:
@camierjs @v-dobrev , if both umpire and raja need -lcamp is this right right way to add the flag?

Reply:
When `CAMP_DIR` is defined (i.e. RAJA/UMPIRE use external CAMP) then we need something like this before `-lcamp`: `$(XLINKER)-rpath,$(CAMP_DIR)/lib -L$(CAMP_DIR)/lib`.

It is probably best to define
```
CAMP_LIB = -lcamp
ifdef CAMP_DIR
   CAMP_LIB = $(XLINKER)-rpath,$(CAMP_DIR)/lib -L$(CAMP_DIR)/lib -lcamp
endif
```
and then use just `$(CAMP_LIB)` in `RAJA_LIB` and `UMPIRE_LIB`.

Reply:
I just pushed up the adjustments